src/Lykke.MatchingEngine.Connector/Models/MeNewTransferModel.cs

```diff
@@ -30,20 +30,23 @@ public class MeNewTransferModel
         public static MeNewTransferModel Create(string id, string fromClientId,
             string toClientId, string assetId, double amount, string feeClientId, double feeSizePercentage)
         {
+            var feeAbsolute = (double)((decimal)amount * (decimal)feeSizePercentage);
```

Comment:
Why do we need double conversion here? double -> decimal -> double

Reply:
When doubles are multiplied 0.035 x 100 result looks like 3.500000000000000004. With decimal representation we have no such problem, but everywhere double is used, so I decided to convert to decimal to avoid such kind of problems.